prow/config/presubmits.yaml

```diff
@@ -2783,6 +2783,27 @@ presubmits:
           requests:
             cpu: "1"
             memory: 1Gi
+  - name: istio-images_validate-2.2
+    decorate: true
+    always_run: true
+    path_alias: github.com/maistra/istio-images-centos
+    skip_report: false
+    branches:
+      - ^maistra-2.2$
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.2"
+        imagePullPolicy: Always
+        command:
+        - ./create-images.sh
```

Comment:
I don't think this will work because this command will run docker - and this job already runs inside docker.

I think you need to start the container with the `entrypoint` command, (e.g. https://github.com/maistra/test-infra/blob/main/prow/config/presubmits.yaml#L719).